docs/source/03-08-gdpr.md

```diff
@@ -11,9 +11,9 @@ can enable the following plugins:
 - [talk-plugin-local-auth](/talk/plugin/talk-plugin-local-auth) - to facilitate email changes and email association
 - [talk-plugin-profile-data](/talk/plugin/talk-plugin-profile-data) - to facilitate account download and deletion
 
```

Comment:
Awesome thank you @babolivier! Quick thing - you have dupe "apply" - otherwise, looks great.